README.md

```diff
@@ -202,6 +202,29 @@ CashRegister.drawer_position_opened # => CashRegister.where(drawer_position: 0)
 CashRegister.drawer_position_closed # => CashRegister.where(drawer_position: 1)
 ```
 
+### Note about default scopes
+
+FlexibleEnum does not play well with default scopes that reference the enum's column. For example, if you have this class:
```

Comment:
You're kind of throwing FlexibleEnum under the bus here. How about something like "Be careful when using default scopes on FlexibleEnum columns. Since FlexibleEnum provides scopes for enum values, setting a default scope on a FlexibleEnum column will result in conflicts. For example, given this model:"